components/script/dom/htmllinkelement.rs

```diff
@@ -377,6 +387,32 @@ impl HTMLLinkElementMethods for HTMLLinkElement {
     // https://html.spec.whatwg.org/multipage/#dom-link-target
     make_setter!(SetTarget, "target");
 
+    // https://html.spec.whatwg.org/multipage/#dom-link-crossorigin
```

Comment:
This should be using an enumerated getter instead: https://dxr.mozilla.org/servo/rev/124301cf484818a87ab0b3e0c4ef34645be66174/components/script/dom/htmlimageelement.rs#296